test-data/input-cloud-provider.json

```diff
@@ -0,0 +1,34 @@
+{
+  "hosts": {
+    "internal": {
+      "region-a": {
+        "selenium-cloud-a-[1:5].example.com": {
+          "port": 4444,
+          "count": 1
+        }
+      }
+    },
+    "cloud-provider": {
+      "provider-1" : {
+        "cloud-provider-1.com": {
+          "port": 4444,
+          "count": 1,
+          "username": "user1",
```

Comment:
We need to test that password and username are inserted correctly to XML. Here is the test: https://github.com/seleniumkit/sctl/blob/master/cmd/generate_test.go Instead of copy-pasting I would update `test-data/input.json` and `output.xml` and check that this test is passing.

Reply:
All the rest is OK.

Reply:
Ah, makes sense. Updated the tests and added checks for `Username` and `Password`.